src/components/cart/CartItem.js

```diff
@@ -41,17 +45,16 @@ export const CartItem = ({ item }) => {
 					<input
 						type="number"
 						min="1"
-						// value={stateQty}
-						// onChange={(e) => {
-						// 	setStateQty((prev) => Number(e.target.value));
-						// handleCartQty(item.id, stateQty);
-						// }}
+						value={stateQty}
+						onChange={(e) => {
+							handleQtyChange(e, item.id, stateQty);
+						}}
 					/>
 				</CartInfoStyle>
 
 				<CartDeleteButtonStyle
 					primary
-					// onClick={() => handleCartDelete(item.id)}
+					onClick={() => handleCartDelete(item.id)}

```

Comment:
The same situation. Should take item.id from props. And don't pass it to function